Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -217,17 +217,17 @@ public class MapboxNavigationService: NSObject, NavigationService {
     /**
      Intializes a new `NavigationService`. Useful convienence initalizer for OBJ-C users, for when you just want to set up a service without customizing anything.
      
-     - parameter route: The route to follow.
-     - parameter routeIndex: The index of the route within the original `RouteController` object.
+     - parameter routeResponse: `RouteResponse` object, containing selection of routes to follow.
+     - parameter routeIndex: The index of the route within the original `RouteResponse` object.
```

Comment:
`routeOptions` isn’t documented, but I wonder if we still need it, because RouteResponse has an `options` property.

Reply:
This argument serves as specification that it handles `RouteOptions` instead of possible `MatchOptions`.